src/vmm/src/device_manager/mmio.rs

```diff
@@ -448,6 +448,13 @@ impl MMIODeviceManager {
                     // Any in-flight packets or events are simply lost.
                     // Vsock is restored 'empty'.
                 }
+                TYPE_RNG => {
+                    let entropy = virtio.as_mut_any().downcast_mut::<Entropy>().unwrap();
+                    if entropy.is_activated() {
+                        info!("kick entropy {id}.");
+                        entropy.process_virtio_queues();
+                    }
+                }
```

Comment:
I feel like this entire `match` block can be replaced by proper dynamic dispatch if we just put `is_actived` and `process_virtio_queues` into the `VirtioDevice` trait, but that's outside the scope of this PR